test/quantization/core/experimental/test_float8.py

```diff
@@ -242,6 +242,23 @@ def compare_binary_with_decimal(binary, decimal, number_name, dtype, device):
         for number in SPECIAL_NUMBERS[dtype]:
             compare_binary_with_decimal(*number, dtype, device)
 
+    @dtypes(*FLOAT8_DTYPES)
```

Comment:
Do this include all 4 of them?

Reply:
yup, well 4 on cpu, and the two cuda specific ones on cuda